setup.py

```diff
@@ -8,13 +10,18 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-with open(os.path.join(here, "README.rst")) as f:
-    README = f.read()
 
-with open(os.path.join(here, "CHANGES.txt")) as f:
-    CHANGES = f.read()
+def read_file(filename):
+    """Open a related file and return its content."""
+    with codecs.open(os.path.join(here, filename), encoding='utf-8') as f:
+        content = f.read()
+    return content
 
-with open(os.path.join(here, "dev-requirements.txt")) as f:
+README = read_file(os.path.join(here, "README.rst"))
+CHANGES = read_file(os.path.join(here, "CHANGES.txt"))
+
+with codecs.open(os.path.join(here, "dev-requirements.txt"),
+                 encoding='utf-8') as f:
```

Comment:
Given that this is identical to the logic in `read_file`, I wonder if the appropriate abstraction is:

```
def open_file(fileame):
    return codecs.open(os.path.join(here, filename), encoding='utf-8')

with open_file('README.rst') as f:
    REDME = f.read()

...etc...
```